src/index.ts

```diff
@@ -60,7 +59,6 @@ async function runAnalysis() {
     'ci',
     '--keyring',
     trustedKeys,
-    '--secret',
```

Comment:
This is the only line changed by the PR. The rest is pre-existing auto-formatting issues.